gpflow/expectations/squared_exponentials.py

```diff
@@ -159,76 +159,118 @@ def _E(p, mean, _, kernel, inducing_variable, nghp=None):
     )
 
 
-@dispatch.expectation.register(
-    (Gaussian, DiagonalGaussian),
-    kernels.SquaredExponential,
-    InducingPoints,
-    kernels.SquaredExponential,
-    InducingPoints,
-)
+@dispatch.expectation.register((Gaussian, DiagonalGaussian),
```

Comment:
you'll have to run `make format` to make the formatting checks pass in the tests.